addons/mail/static/src/js/services/chat_manager.js

```diff
@@ -1115,20 +1114,93 @@ var ChatManager =  AbstractService.extend({
             self.mentionPartnerSuggestions = result.mention_partner_suggestions;
             self.discussMenuID = result.menu_id;
 
-            // Shortcodes: canned responses and emojis
+            // Shortcodes: canned responses            
             _.each(result.shortcodes, function (s) {
-                if (s.shortcode_type === 'text') {
-                    self.cannedResponses.push(_.pick(s, ['id', 'source', 'substitution']));
-                } else {
-                    self.emojis.push(
-                        _.pick(s, ['id', 'source', 'unicode_source', 'substitution', 'description'])
-                    );
-                    self.emojiSubstitutions[_.escape(s.source)] = s.substitution;
-                    if (s.unicode_source) {
-                        self.emojiSubstitutions[_.escape(s.unicode_source)] = s.substitution;
-                        self.emojiUnicodes[_.escape(s.source)] = s.unicode_source;
-                    }
-                }
+                self.cannedResponses.push(_.pick(s, ['id', 'source', 'substitution']));
+            });
+            // emojis
+            self.emojiUnicodes={
+                ":)":"😊",
+                ":D":"😃",
+                "xD":"😆",
+                ":-)":"😂",
```

Comment:
I suggest `x'D` for this one

Reply:
@Elkasitu oh please no, everybody uses xD

Reply:
for the crying one, I mean :innocent:

Reply:
Yes good idea i think the :-) should be an alternative for :)

Reply:
Yes, indeed. :)